src/frontend/src/scheduler/plan_fragmenter.rs

```diff
@@ -380,6 +381,7 @@ impl StageGraphBuilder {
 
 impl BatchPlanFragmenter {
     /// Split the plan node into each stages, based on exchange node.
+    #[instrument(skip_all)]
```

Comment:
I'd suggest always manually create spans instead of using `instrument` macro.

Reply:
Could you elaborate more on this?

Reply:
See https://github.com/risingwavelabs/risingwave/pull/4576, use tracing::trace_span to create spans, and use `future.instrument(span)` to monitor futures.

Reply:
IIUC they are same, why we can't use `macro`?

Reply:
I feel like it would be easier to control spans manually. Otherwise there'll be too many unwanted spans. Also sometimes the span doesn't always have the same lifetime as functions. For example, spans in the streaming executors can live more than a `next` call.

Reply:
But it's up to you to choose the way you want to use. Probably in some cases it's easier to attach `instrument` everywhere.

Reply:
Yes, batch execution has different symatics as streaming executor, so the choice maybe different.